includes/elements/container.php

```diff
@@ -443,25 +510,31 @@ protected function register_container_layout_controls() {
 			]
 		);
 
+		$is_container_grid_active = Plugin::$instance->experiments->is_feature_active( 'container_grid' );
+
+		$this->add_control(
+			'container_type',
+			$this->get_container_type_control_options( $is_container_grid_active )
+		);
```

Comment:
@udidol Because we removed the initial `--display` property in the CSS and need to set it from the Container Type control now.

https://github.com/elementor/elementor/pull/21478/files#diff-ea32535ec65ff78a24fe64a87ca2fe2b8c299264dff60ea195ca0dbfd803cb82L4

But we cannot in the CSS conditionally check if the Experiment is active and decide whether or not have that CSS property. So I added a hidden control to set the `--display` to `flex` from there if the Experiment is OFF.

This can be removed when the Experiment is merged.